### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by ecrispenscd**

### CODE REVIEW

**Review by cs-shravan-konthalapally — CHANGES_REQUESTED** [review]
change the folder name to itglue.

File: ITGlue/health_check.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def check(config):
+    # TODO: implement health check for the connector here
```
Comment (cs-shravan-konthalapally) [inline ITGlue/health_check.py:2]: Verify the configuration using the API key from configuration

File: ITGlue/itglueapi.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        headers = {
+            'x-api-key': self.api_key,
+            'Content-Type': 'application/json'
+        }
+
+        # Build URL
+        url = self.host + endpoint
+
+        # Make request
+        response = request(method='get', url=url, params=params, headers=headers)
```
Comment (cs-shravan-konthalapally) [inline ITGlue/itglueapi.py:53]: Add logging and exception handling

**Review by cs-shravan-konthalapally — CHANGES_REQUESTED** [review]
Add the logger and exception handling/change the folder structure

**Comment by cs-prerna-joshi** [comment]
Integration Team will take care of the changes required.

**Pull request merged**